README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@bradenmacdonald That can be a useful intermediary step, but the goal is to use the concrete use cases to push the formalization, and get to a continuous integration workflow. The end goal is to not require UI at all for updates - that once we are done, pressing "merge" on a PR on github is all that is required to get the production instance to update.